packages/create-redwood-app/src/telemetry.js

```diff
@@ -1,6 +1,115 @@
-const { sendTelemetry } = require('@redwoodjs/telemetry/dist/sendTelemetry')
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api'
+import opentelemetry, { SpanStatusCode } from '@opentelemetry/api'
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http'
+import { Resource } from '@opentelemetry/resources'
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node'
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+import envinfo from 'envinfo'
+import system from 'systeminformation'
 
-// if this script is run directly by node then telemetry will be sent in immediately
-;(async function () {
-  await sendTelemetry()
-})()
+/**
+ * @type NodeTracerProvider
+ */
+let traceProvider
+
+/**
+ * @type BatchSpanProcessor
+ */
+let traceProcessor
+
+/**
+ * @type OTLPTraceExporter
+ */
+let traceExporter
+
+/**
+ * @type Span
+ */
+let rootSpan
+
+export async function startTelemetry() {
+  // Logger
+  // TODO: This line should be removed or at least the log level raised after experimentation
+  diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO)
+
+  // Resources
+  const info = JSON.parse(
+    await envinfo.run(
+      {
+        System: ['OS', 'Shell'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        npmPackages: '@redwoodjs/*',
+        IDEs: ['VSCode'],
+      },
+      { json: true }
+    )
+  )
+
+  // get shell name instead of path
+  const shell = info.System?.Shell // Windows doesn't always provide shell info, I guess
+  if (shell?.path?.match('/')) {
+    info.System.Shell.name = info.System.Shell.path.split('/').pop()
+  } else if (shell?.path.match('\\')) {
+    info.System.Shell.name = info.System.Shell.path.split('\\').pop()
+  }
+  const cpu = await system.cpu()
+  const mem = await system.mem()
+
+  const resource = Resource.default().merge(
+    new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: 'create-redwood-app',
+      // TODO: Get a better way of recording what version of CRWA is running because this is just null...
+      [SemanticResourceAttributes.SERVICE_VERSION]:
+        info.npmPackages != null
+          ? info.npmPackages['@redwoodjs/core']?.installed
+          : undefined,
```

Comment:
Any reason you can't just get it from this package's `package.json`? Like in `create-redwood-app.js`

https://github.com/redwoodjs/redwood/blob/8569f5ff0d5996c13aa0b23194aed39a54acf9c0/packages/create-redwood-app/src/create-redwood-app.js#L22

Reply:
I'd been using that technique in my earlier experimenting and it worked fine. I only replaced it with a to-do because vscode was yelling at me. I'll happily add it back.

Reply:
Ah gotcha—yelling at you with red squiggles?

Reply:
Yeah but it's fine now. I must have been importing it with the extension or something before.